### PR TITLE
prep-css-syntax | Chris Kim

### PR DESCRIPTION
**Pull request by chris-ys-kim**
Screen Shot 2021-04-20 at 8 27 25 PM

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Chris! Not too sure what happened, but your files were not added to this branch! 😟 
https://github.com/chris-ys-kim/prep-0421-code-solutions/pull/7/files

**Review by thebearingedge — COMMENTED** [review]
Hi Chris! Please wait until you get a review before you merge your Pull Request

**Comment by chris-ys-kim** [comment]
oh my bad. I'll wait next time and it won't happen again.    On Tuesday, April 20, 2021, 09:20:19 PM PDT, Tim Davis ***@***.***> wrote:  
 
 


@thebearingedge commented on this pull request.

Hi Chris! Please wait until you get a review before you merge your Pull Request

**Pull request merged**
